src/rdf4cpp/Timezone.hpp

```diff
@@ -130,23 +132,144 @@ struct Timezone {
 
 using OptionalTimezone = std::optional<Timezone>;
 
-using TimePoint = std::chrono::time_point<std::chrono::local_t, std::chrono::milliseconds>;
+template<typename I = int64_t>
+struct Year {
```

Comment:
I think it would probably make sense to clone as much of the interface of `std::chrono::year` as possible, as this type is part of the API surface